gazebo/rendering/GpuLaser.cc

```diff
@@ -427,59 +439,67 @@ GpuLaser::DataIter GpuLaser::LaserDataBegin() const
 {
   const unsigned int index = 0;
   // Data stuffed into three floats (RGB)
-  const unsigned int skip = 3;
+  unsigned int skip = 3;
   // range data in R channel
   const unsigned int rangeOffset = 0;
   // intensity data in G channel
   const unsigned int intenOffset = 1;
-  return DataIter(index, this->dataPtr->laserBuffer.data(), skip, rangeOffset,
-      intenOffset, this->dataPtr->w2nd);
+  return {index, this->dataPtr->laserBuffer.data(), skip, rangeOffset,
+          intenOffset, this->dataPtr->horizontalRangeCount};
 }
 
 //////////////////////////////////////////////////
 GpuLaser::DataIter GpuLaser::LaserDataEnd() const
 {
-  const unsigned int index = this->dataPtr->h2nd * this->dataPtr->w2nd;
+
+  unsigned int index;
+  index = this->dataPtr->verticalRangeCount * this->dataPtr->horizontalRangeCount;
 
   // Data stuffed into three floats (RGB)
-  const unsigned int skip = 3;
+  unsigned int skip = 3;
```

Comment:
Similar here, both can be const/constexpr, no need to assign `index` in next line.